app/services/ocrService.py

```
@@ -19,25 +20,100 @@
+    def is_client_error(self):
+        return self._is_client_error
+
+
+# 사용자별 OCR 이미지 경로 생성
+def get_user_ocr_path(user_hash: str, filename: str) -> str:
+    return f"users/{user_hash}/ocr/{filename}"
+
+
+def _get_bucket():
+    if not GCS_BUCKET_NAME:
+        raise OcrError("GCS 버킷 설정이 올바르지 않습니다.")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)